openfga/v1/errors_ignore.proto

```diff
@@ -67,7 +67,7 @@ enum ErrorCode {
   exceeded_entity_limit = 2053;
   invalid_contextual_tuple = 2054;
   duplicate_contextual_tuple = 2055;
-  invalid_relation_type = 2056;
+  invalid_authorization_model = 2056;
```

Comment:
We are keeping the same error code for a completely different error. Ideally, we should not be removing errors just adding.

This ensures that regardless of API version, users do not receive error codes that may mean something completely different on another version.

Reply:
I added this error code to use in my pr https://github.com/openfga/openfga/pull/223, and now I am not using it anymore. These changes are the downside of having the protos stored in a different repo. Where else are we using it?